pyat/at/lattice/elements/basic_elements.py

```diff
@@ -626,5 +627,35 @@ def __init__(self, family_name: str, energy_loss: float, **kwargs):
         kwargs.setdefault("PassMethod", self.default_pass[False])
         super().__init__(family_name, EnergyLoss=energy_loss, **kwargs)
 
+class Feedback(Element):
+    """Transverse and Longitudinal Feedback element"""
+
+    def __init__(self, family_name: str,
+                       GX: float = 0.0, 
```

Comment:
capital letter variables do not fullfil PEP8 requirements I think. 
Please check PEP8 and use black for code formatting.